src/Compatibility/Core/src/iOS/Renderers/FrameRenderer.cs

```diff
@@ -136,7 +136,6 @@ public override void LayoutSubviews()
 			if (_previousSize != Bounds.Size)
 			{
 				SetNeedsDisplay();
-				this.UpdateBackgroundLayer();
```

Comment:
Could include an UITest? Should be a VerticalStackLayout including a Frame and a Border with gradients background and changing the View size at runtime. If the gradients renders correctly after change the size, the test pass. To change the size, can use something like a Button. And to verify the rendering, could use the `VerifyScreenshot` method.


Let me know if needs help or want myself to include it.

Reply:
@jsuarezruiz I've added the UI test as requested, feel free to add any additional test if you feel it's needed.